docs/book/src/introduction/installation.md

```diff
@@ -2,35 +2,91 @@
 
 The _Sway toolchain_ is sufficient to compile Sway smart contracts. Otherwise, note that if you want to run Sway smart contracts (e.g. for testing), a Fuel Core full node is required, which is packaged together with the _Sway toolchain_ together as the _Fuel toolchain_.
 
-## Installing from Pre-compiled Binaries
+## Install via Pre-compiled Binaries (Recommended)
 
-Pre-compiled release binaries for Linux and macOS are available for the Sway toolchain. Native Windows is currently unsupported ([tracking issue for Windows support](https://github.com/FuelLabs/sway/issues/1526)). Windows Subsystem for Linux should work but is not officially supported.
+Installing via pre-compiled release binaries is the recommended way to get up and running with the Sway toolchain. Pre-compiled binaries for Linux and macOS are available. Native Windows is currently unsupported ([tracking issue for Windows support](https://github.com/FuelLabs/sway/issues/1526)). Windows Subsystem for Linux should work but is not officially supported.
 
 [`fuelup`](https://github.com/FuelLabs/fuelup) is the equivalent of Rust's `rustup` for the Fuel toolchain. It enables easily downloading binary releases of the Fuel toolchain.
 
-Start by installing `fuelup` with:
+1. Start by installing `fuelup` with the following command:
 
-```sh
-curl --proto '=https' --tlsv1.2 -sSf \
-    https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
-```
+   ```sh
+   curl --proto '=https' --tlsv1.2 -sSf \
+   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
+   ```
 
-`fuelup-init` will ask for permission to add `~/.fuelup/bin` to your PATH. Otherwise, you can also pass `--no-modify-path` so that `fuelup-init` does not modify your PATH:
+   This downloads the `fuelup-init` script to a temp directory on your machine, which installs `fuelup`. `fuelup-init` will ask for permission to add `~/.fuelup/bin` to your PATH. Otherwise, you can also pass `--no-modify-path` so that `fuelup-init` does not modify your PATH:
 
-```sh
-curl --proto '=https' --tlsv1.2 -sSf \
-    https://fuellabs.github.io/fuelup/fuelup-init.sh | sh -s -- --no-modify-path
-```
+   ```sh
+   curl --proto '=https' --tlsv1.2 -sSf \
+   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh -s -- --no-modify-path
+   ```
 
-Once `fuelup` is installed, `fuelup-init` automatically runs the command below
+2. Once fuelup is installed, fuelup-init automatically runs `fuelup toolchain install latest` to install the latest toolchain
 
-```sh
-fuelup toolchain install latest
-```
+   You can run `fuelup toolchain install latest` at anytime to get the most up-to-date toolchain.
```

Comment:
```suggestion
   You can run `fuelup update` at anytime to get the most up-to-date toolchain.
```

I'd prefer to encourage `fuelup update` here, which updates all official toolchains that a user has installed, although `fuelup toolchain install latest` also works